deployer/src/handlers/mod.rs

```diff
@@ -493,6 +494,43 @@ pub async fn delete_deployment(
     }
 }
 
+#[instrument(skip_all, fields(%project_name, %deployment_id))]
+#[utoipa::path(
+    put,
+    path = "/projects/{project_name}/deployments/{deployment_id}",
+    responses(
+        (status = 200, description = "Started a specific deployment.", body = shuttle_common::models::deployment::Response),
+        (status = 500, description = "Database or streaming error.", body = String),
+        (status = 404, description = "Could not find deployment to start", body = String),
+    ),
+    params(
+        ("project_name" = String, Path, description = "Name of the project that owns the deployment."),
+        ("deployment_id" = String, Path, description = "The deployment id in uuid format.")
+    )
+)]
+pub async fn start_deployment(
+    Extension(persistence): Extension<Persistence>,
+    Extension(deployment_manager): Extension<DeploymentManager>,
+    Extension(claim): Extension<Claim>,
+    Path((project_name, deployment_id)): Path<(String, Uuid)>,
+) -> Result<()> {
+    if let Some(deployment) = persistence.get_runnable_deployment(&deployment_id).await? {
```

Comment:
Given we have a deployment per service, I think we could have assumed that we can start the latest deployment as well, but only if it's `Stopped` or `Running`. I wonder if you thought about sending the `deployment_id` from the gateway because of the possibility of having a deployment per service, in a workspace-based Shuttle project.

Reply:
Yes, but then we won't be able to use this for a `cargo shuttle start` command

Reply:
We want the idle logic to be removed from the deployer and moved to the gateway, so the gateway will decide what deployment to start, leaving just the deployer as a layer of managing runtimes and storing state/exposing state.